src/components/TextField/TextField.jsx

```diff
@@ -99,7 +99,9 @@ const focusVariant = variant({
 const Wrapper = styled(Box)`
   ${errorVariant}
   ${disabledVariant}
-  width: fit-content;
+  width: 100%;
```

Comment:
Acho que o comportamento do `fit-content` é um pouco diferente do `100%`. Dei uma pesquisada, e pelo que eu vi, o fit-content só vai pegar, no mínimo o espaço do proprio conteúdo e no máximo o espaço disponível "por fora", já o `100%` vai pegar sempre a largura maxima do que está por fora. Pensando na implementação do TextField em layout's com vários componentes por linha, acho que isso pode ser um pouco prejudicial. O que acha? 🚀

Me baseei nesses link's para entender o `fit-content`:
https://stackoverflow.com/questions/30704073/what-is-the-difference-between-css-fit-content-and-max-content
https://developer.mozilla.org/pt-BR/docs/Web/CSS/width
https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content

Reply:
Para ficar documentado, conversei com o @dpnunez sobre o contexto desta change. Basicamente sobre o Width foi em uma mudança que fizemos anteriormente no componente e resolvermos voltar atrás.